### PR TITLE
Fixed plot_plot bug and change vprint usage 

### PR DESCRIPTION
**Pull request by miguelglezb**
The following changes were made to fix issues when running old and recent tutorials:

- PR #437 had an issue due to the change I made from plot_plot to quick_plot in @angelrls  tutorials. They were stuck with the old name. All instances with `plot_plot` are now named `quick_plot`.
- `vprint` is an unbound function, but in class `SkyFromObject` was used as an attribute. This was causing an issue when running `py.test` and showed a warning in `pylance`. While the execution in VS code was not showing issues, the function should not be run as attribute in this case anyway.
- Changed the condition for verbose in `vprint` due to error when the condition was False. Instead of  
```python
if kwargs['verbose']:
```
it is now 
```python
if 'verbose' in kwargs:
```

### CODE REVIEW

File: src/pykoala/ancillary.py [hunk 1]
```diff
@@ -28,10 +28,9 @@ def vprint(*arg, **kwargs):
     """
     Prints the arguments only if verbose=True.
     """
-    if kwargs['verbose']:
+    if 'verbose' in kwargs:
```
Comment (paranoya) [inline src/pykoala/ancillary.py:31]: Shouldn't we check that `verbose` is `True`?
```python
    if 'verbose' in kwargs and verbose is True:
```
or
```python
    if kwargs.get('verbose', False):
```

File: src/pykoala/corrections/sky.py [hunk 2]
```diff
@@ -826,10 +827,10 @@ def __init__(self, dc, bckgr_estimator='mad', bckgr_params=None,
         cont_estimator_args : dict, optional
             Arguments for the continuum estimator. Default is None.
         """
-        self.vprint("Creating SkyModel from input Data Container")
+        vprint(self,"Creating SkyModel from input Data Container")
```
Comment (paranoya) [inline src/pykoala/corrections/sky.py:830]: Why do you need to pass `self` as an argument?

File: src/pykoala/ancillary.py (hunk 1 above)
Comment (miguelglezb) [reply src/pykoala/ancillary.py:31]: `'verbose'` is just a string, not a boolean. The old line asks if `kwargs['verbose']` is `True`, not if 'verbose' is `True`, which would be a nonsense thing to ask. The new line (whether or not 'verbose' is an element in the list of keys `kwargs`)  states the same condition without spitting out a `KeyError`

File: src/pykoala/corrections/sky.py (hunk 2 above)
Comment (miguelglezb) [reply src/pykoala/corrections/sky.py:830]: As stated above: 

`vprint` is an unbound function, but in class `SkyFromObject` was used as an attribute. This was causing an issue when running `py.test` and showed a warning in `pylance`. While the execution in VS code was not showing issues, the function should not be run as attribute in this case anyway. For an attribute, it makes sense to have the original line, but when using an external function (defined outside `src/pykoala/corrections/sky.py`)

**Pull request merged**